src/SFA.DAS.PR.Application/AccountProviderLegalEntities/Queries/GetAccountProviderLegalEntities/GetAccountProviderLegalEntitiesQueryValidator.cs

```diff
@@ -0,0 +1,35 @@
+﻿using FluentValidation;
+
+namespace SFA.DAS.PR.Application.AccountProviderLegalEntities.Queries.GetAccountProviderLegalEntities;
+public class GetAccountProviderLegalEntitiesQueryValidator : AbstractValidator<GetAccountProviderLegalEntitiesQuery>
+{
+    public const string UkprnAccountHashIdValidationMessage = "Either one of Ukprn or AccountHashId is required.";
+    public const string UkprnFormatValidationMessage = "Currently a Ukprn must start with the value 1 and should be 8 digits long.";
+    public const string OperationFilterValidationMessage = "Currently at least one operation filter must to be supplied.";
+    public const string OperationFilterFormatValidationMessage = "Operation filter values are limited to 0, 1 or 2.";
+    public GetAccountProviderLegalEntitiesQueryValidator()
+    {
+        RuleFor(x => x.Ukprn)
+            .Must(Ukprn => Ukprn.HasValue)
```

Comment:
We used NotEmpty before, is that not an option any more, this should check for both null and zero so there is no need for a `When` condition

Reply:
@DevenKShah The when in this instance is used to validate the opposite parameter in the request. Ukprn must have a value when AccountHashId is empty and vice versa.